File: FileRenamer/ViewModels/Controls/About/ReferenceViewModel.cpp
#include "pch.h"
#include "ReferenceViewModel.h"
#include "ReferenceViewModel.g.cpp"

namespace winrt::FileRenamer::implementation
{
	ReferenceViewModel::ReferenceViewModel()
	{
		_referenceDict.Append(winrt::make<ReferenceKeyValuePairModel>(L"Microsoft.Windows.CppWinRT", L"https://github.com/Microsoft/cppwinrt"));
		_referenceDict.Append(winrt::make<ReferenceKeyValuePairModel>(L"Microsoft.Windows.SDK.BuildTools", L"https://aka.ms/WinSDKProjectURL"));
		_referenceDict.Append(winrt::make<ReferenceKeyValuePairModel>(L"Mile.Xaml", L"https://github.com/ProjectMile/Mile.Xaml"));
	}

	winrt::WinrtCollections::IObservableVector<winrt::FileRenamer::ReferenceKeyValuePairModel> ReferenceViewModel::ReferenceDict()
	{
		return _referenceDict;
	}
}

File: FileRenamer/ViewModels/Controls/About/ReferenceViewModel.cs
using FileRenamer.Models.Controls.About;
using System.Collections.Generic;

namespace FileRenamer.ViewModels.Controls.About
{
    public sealed class ReferenceViewModel
    {
        //项目引用信息
        public List<ReferenceKeyValuePairModel> ReferenceDict = new List<ReferenceKeyValuePairModel>()
        {
            new ReferenceKeyValuePairModel() {Key = "Microsoft.Windows.SDK.BuildTools" ,Value = "https://aka.ms/WinSDKProjectURL" },
            new ReferenceKeyValuePairModel() {Key = "Mile.Xaml" ,Value = "https://github.com/ProjectMile/Mile.Xaml" },
        };
    }
}
